lib/httpRequestBuilder.js

```diff
@@ -60,7 +60,8 @@ function requestBuilder (defaults) {
     }
 
     if (bodyBuf && bodyBuf.length > 0) {
-      headers['Content-Length'] = '' + bodyBuf.length
+      const uuidCount = (bodyBuf.toString().match(/\[<uuid>\]/g) || []).length
+      headers['Content-Length'] = `${bodyBuf.length + (uuidCount * 28)}`
```

Comment:
Sure, I will absolutely do that, I think in my haste to respond to the earlier comment I forgot to go back over the maths. I'd gotten the earlier figure by comparing the content length with the tag and with the UUID, but I will look into this and add a test as you suggested below. Should have time to work on this tonight or tomorrow.